tests/integrationtest/r/executor/partition/global_index.result

```diff
@@ -231,5 +232,6 @@ bbb
 bbc
 explain format = 'brief' select a from t partition (p1) order by a;
 id	estRows	task	access object	operator info
-IndexReader	3.00	root	partition:p1	index:IndexFullScan
-└─IndexFullScan	3.00	cop[tikv]	table:t, index:a(a)	keep order:true
+IndexReader	3.00	root	partition:p1	index:Selection
+└─Selection	3.00	cop[tikv]	NULL	in(_tidb_pid, pid1)
```

Comment:
what is the pid1 named for? how about use p-name

Reply:
It's replaced by mysql-tester, in the original plan, it always be a number, like '129', '130'.

Reply:
got